File: StrategyPattern/Bank/C++/NormalAccount.cpp
#include  "NormalAccount.h"

void NormalAccount::display()
{

	cout << "This is Normal account" << endl;
	cout << "Balance  : " << balance << endl;
	cout << "Account openning : " << openningDate.tm_year+ 1900 << "/" <<openningDate.tm_mon <<"/" <<
		openningDate.tm_mday <<" " << openningDate.tm_hour << ":" << openningDate.tm_min <<":"<< openningDate.tm_sec<< endl;
}

File: StrategyPattern/Bank/JAVA/NormalAccount.java
package ACCOUNT;

import java.util.Date;

public class NormalAccount extends Account{

	public NormalAccount()
	{
		transaction = new NormalWithdraw(balance);
	}
	
	@Override
	void display() {
		System.out.println("This is Normal account");
		System.out.println("Balance  : " + balance);
		System.out.println("Account openning : " + openningDate.toString());
	}

}
